packages/nimble-components/src/combobox/template.ts

```diff
@@ -10,7 +16,9 @@ import type { Combobox } from '.';
 import { anchoredRegionTag } from '../anchored-region';
 import { DropdownPosition } from '../patterns/dropdown/types';
 import { overflow } from '../utilities/directive/overflow';
+import { filterNoResultsLabel } from '../label-provider/core/label-tokens';
 
+/* eslint-disable @typescript-eslint/indent */
```

Comment:
Can you make sure all these `eslint-disable` have a bookend `eslint-enable`

Reply:
Got everything in `nimble-components`, I think.

Reply:
I request it for refactoring purposes / modifications. Disable for a block of a file, not the rest of the file as stuff can be inserted later / moved around that may or may not intentionally be found.

> This conversation needn't block this PR so I left it Resolved.

@jattasNI @m-akinc this makes it very easy to miss since GitHub notifications are per PR not per mention (at least for me). I'd prefer the PR owner keep them open till merge and then re-open them so the open conversations are discoverable.

Reply:
>  I'd prefer the PR owner keep them open till merge and then re-open them...

To confirm, you're not asking for a real change in how we've been operating--just that the author resolves threads when the resolving code gets pushed, then a subsequent commenter on a thread should unresolve it. Right?

Reply:
Right. Whoever merges the PR can temporarily resolve the unresolved, non-blocking comments to merge but should make sure they are re-opened (and ideally that they are temporarily resolved close to the merge, not significantly in advance).